.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: python
 python:
   - '2.7'
-before_install:
```

Comment:
We can try this, but it made builds fail in the past: see b2401a7ee76b951d254e7cfbf8eefd87e3a2ee41.